lib/brainstem/presenter_collection.rb

```diff
@@ -226,6 +226,11 @@ def handle_only(scope, only)
 
     def run_filters(scope, options)
       extracted_filters = extract_filters(options)
+
+      params = extracted_filters.each.with_object({}) do |(key, val), hash|
+        hash[key] = val[:arg]
```

Comment:
I think there may be situations where you want to see all of the requested filters, even ones without an arg, which is why I wasn't checking for nil there.